test/schedule/fetcher_test.exs

```diff
@@ -57,6 +57,7 @@ defmodule Schedule.FetcherTest do
           end
         )
 
+      assert log =~ "Sent updated schedule data to receiving process"
```

Comment:
**suggestion**: this is fine as far as it goes, but we could make a stronger assertion: `assert log =~ /Sent updated schedule data to receiving process, time_in_ms=\d+/`